whist/core/util.py

```diff
@@ -0,0 +1,16 @@
+"""
+Collection of utility functions.
+"""
+
+
+def enforce_str_on_dict(dictionary: dict, keys: set[str]) -> dict:
+    """
+    Forces a dictionary to use string values instead of objects for the given keys.
+    :param dictionary: which needs transformation
+    :param keys: that values needed to be transformed
+    :return: the above dictionary with values changes to string
+    """
+    for key in keys:
+        if key in dictionary:
+            dictionary[key] = str(dictionary[key]) if dictionary[key] else None
+    return dictionary
```

Comment:
change type hint for arg from set to iterable or something.

Reply:
Just following pydantic standards which uses sets